apps/docs/src/app/pages/utils/pipes/has-observers/demos/has-observers-component/has-observers.demo.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { HasObserversPipe } from '@ngx/utils';
+
+@Component({
+	standalone: true,
+	imports: [HasObserversPipe],
+	templateUrl: './has-observers.demo.component.html',
+	selector: 'has-observers-demo-component',
+})
+export class HasObserversPipeDemoComponent {
+	@Output() public somethingHasCompleted: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
Just fyi and not obrigatory, but properties like these can be `readonly` because the can't and shouldn't be overwritten at any point.